emu/src/arm7tdmi.rs

```diff
@@ -316,45 +325,156 @@ mod tests {
     #[test]
     fn check_block_data_transfer() {
         {
-            // STR
-            let op_code = 0b1110_1000_1011_1101_1000_0000_0000_0111;
+            // LDM with post-increment
+            let op_code = 0b1110_100_0_1_0_1_1_1101_0000000010100010;
```

Comment:
Don't understand why formatting the number in this way? Which part you want highlight?

Reply:
To me it's a bit easier when you want to understand what this instruction does. For example in this case first 4 bits are cond, then 3 bits of opcode, 1 bit of P, 1 bit of U and so on...

Reply:
If you group every 4 I find it harder to see which bit does what